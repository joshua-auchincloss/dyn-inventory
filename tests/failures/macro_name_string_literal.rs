// macro_name must be an identifier, not a string literal
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    TraitY: PluginY<T> {
        pub a: u8,
        t: T,
    };
    macro_name = "my_macro",
);

fn main() {}
