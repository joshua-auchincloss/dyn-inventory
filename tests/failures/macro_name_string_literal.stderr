error: expected identifier
  --> tests/failures/macro_name_string_literal.rs:18:18
   |
18 |     macro_name = "my_macro",
   |                  ^^^^^^^^^^
